portfolio/src/main/webapp/aboutme_script.js

```diff
@@ -14,10 +14,10 @@
 
 /** Creates a map and adds it to the page. */
 function createMap() {
-
+    const homeCoordinates = {lat: 40.821520, lng: -73.955770}
     const map = new google.maps.Map(
       document.getElementById('map'),
-      {center: {lat: 40.821520, lng: -73.955770}, zoom: 16,
+      {center: homeCoordinates, zoom: 16,
```

Comment:
Some formatting issues here and below. No line break before the opening brace, so the bracket before center can go on the previous line. Same with the styles below. More info on general formatting here: https://google.github.io/styleguide/jsguide.html#formatting-nonempty-blocks